src/items/traits.md

```diff
@@ -95,6 +95,99 @@ Object safe traits can be the base trait of a [trait object]. A trait is
 * It must not have any associated constants.
 * All supertraits must also be object safe.
 
+When there isn't a `Self: Sized` bound on a method, the type of a method
+receiver must be one of the following types:
+
+* `&Self`
+* `&mut Self`
+* [`Box<Self>`]
+* [`Rc<Self>`]
+* [`Arc<Self>`]
+* [`Pin<P>`] where `P` is one of the object safe receiver types
```

Comment:
This is the first time the phrase  "object safe receiver types" has been mentioned. Consider possibly "where `P` is one of the types above" or similar.

Reply:
Good point, updated!